docs/includes/reactivestreams/engine.adoc

```diff
@@ -138,7 +138,7 @@ Single.just("1")
 
 |===
 
-=== Operator chains composition
+=== Operator Chains Composition
 
 In the situations when part of the operator chain needs to be prepared in advance,
```

Comment:
In situations where part of the operator chain needs to be prepared in advance, the `compose` and `to` operators can be used.